app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,11 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  if fog_credentials.present?
```

Comment:
This configuration seems a daimon specific config, (not only each uploader)
If so, should it be moved to `ApplicationUploader`, `config/initializers/carrierwave.rb` or somewhere?